kube-client/src/config/incluster_config.rs

```diff
@@ -40,14 +27,6 @@ pub enum Error {
     ParseCertificates(#[source] pem::PemError),
 }
 
-/// Returns Kubernetes address from specified environment variables.
-pub fn kube_server() -> Result<http::Uri, Error> {
-    kube_host_port()
-        .ok_or(Error::MissingEnvironmentVariables)?
-        .parse::<http::Uri>()
-        .map_err(Error::ParseClusterUrl)
-}
-
 pub fn kube_dns() -> http::Uri {
```

Comment:
`http::Uri::from_static("https://kubernetes.default.svc/")` instead of `.build().unwrap()`?